packages/plugins/Traderxyz/src/apis/nftswap.tsx

```diff
@@ -0,0 +1,12 @@
+import { NftSwap } from '@traderxyz/nft-swap-sdk'
+import { ExternalProvider, Web3Provider } from '@ethersproject/providers'
+import { getNetworkName, useWeb3Provider } from '@masknet/web3-shared-evm'
+
+export function getTraderApi(chainId: number) {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
DON'T DO THIS. Rename this function to `useTraderApi` instead.

Reply:
Sure.

Reply:
done.

Reply:
Done.

Reply:
Please do not disable react-hooks/rules-of-hooks.